scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -89,6 +89,9 @@ struct pcpu_ctx {
 
 struct pcpu_ctx pcpu_ctx[MAX_CPUS];
 
+u64 current_cpu_vtime[MAX_CPUS];
```

Comment:
Maybe add a comment explaining what it's supposed to contain? The `dom_ctx` field too.